src/cmd/hotspots/claim.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{cmd::*, result::Result};
+use bulk_claim_rewards::{claim_rewards::claim_rewards_blocking, claim_rewards::ClaimRewardsArgs};
+use hpl_utils::dao::{Dao, SubDao};
+use solana_program::pubkey::Pubkey;
+use std::str::FromStr;
+use std::time::Instant;
+
+#[derive(Clone, Debug, clap::Args)]
+pub struct Cmd {
+    #[arg(long)]
+    hotspot_owner: Option<String>,
+    /// Type of rewards to claim, either 'iot' or 'mobile'
+    #[arg(long)]
```

Comment:
Use the same conventions as the `assert` command? i.e. read it more like a sentence without all the long options. Remove arg(long) and make sub_dao the first argument